requirements/workbench.in

```diff
@@ -3,13 +3,10 @@
 
 -r test.txt               # Core and testing dependencies for this package
 
-mysqlclient
-
 fs-s3fs
 lxml
 requests
 pypng
 simplejson
 web-fragments
 webob
-XBlock[django]
```

Comment:
The `workbench` dependencies were used only in integration tests. We can remove `workbench.in` and `workbench.txt` files. Then, we should remove their mentions in `Makefile` and `dev.in`.